apps/script-service/Services/IScriptFunctions.cs

```diff
@@ -7,7 +7,7 @@ interface IScriptFunctions
 
   string? GeneratePdf(string templateId, string filename, object values);
   IDictionary<string, object?>? GetConfiguration(string @namespace, string name);
-  FormDataResult? GetFormData(string formId);
```

Comment:
Are you getting rid of an existing functionality here ?

Reply:
Good catch.  Musta been brain dead that day.